tutorials/v7/draw_rh1_large.cxx

```diff
@@ -49,24 +49,26 @@ void draw_rh1_large()
 
    auto frame = canvas->AddFrame();
 
-   frame->SetGridX(true).SetGridY(true);
-   frame->AttrX().SetZoom(nbins*0.2, nbins*0.8);
+   frame->gridx = true;
+   frame->gridy = true;
+   frame->x.zoommin = nbins*0.2;
+   frame->x.zoommax = nbins*0.8;
```

Comment:
```suggestion
   frame->gridX = true;
   frame->gridY = true;
   frame->x.zoomMin = nbins*0.2;
   frame->x.zoomMax = nbins*0.8;
```
reads way better, IMO...

Reply:
I apply such naming convention to all classes